src/sentry/static/sentry/app/views/settings/organizationRelay/modals/add/terminal.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+
+type Props = {
+  command: string;
+};
+
+const Terminal = ({command}: Props) => (
```

Comment:
Should this be something reusable and live in a storybook?

Reply:
I thought about it, but imo we should only make it reusable when more components will need to use it.

Reply:
I love this look. Maybe we can port this over to getsentry/sentry-docs and getsentry/develop too?

Reply:
I didn't think about it before, but yes let's do it 😀